tests/ignite/metrics/test_average_precision.py

```diff
@@ -50,8 +50,8 @@ def test_input_types():
         ap.update((torch.randint(0, 2, size=(10,)).long(), torch.randint(0, 2, size=(10, 5)).long()))
 
 
-def test_check_shape():
-    ap = AveragePrecision()
+def test_check_shape(available_device):
```

Comment:
No need to add available_device fixture to the tests checking if an error is raised